rest-api-spec/src/main/resources/rest-api-spec/test/scripts/20_get_script_context.yml

```diff
@@ -4,24 +4,20 @@
       reason: "get_all_contexts introduced in 7.6.0"
   - do:
       get_script_context: {}
-  - match: { contexts.aggregation_selector: {} }
-  - match: { contexts.aggs: {} }
-  - match: { contexts.aggs_combine: {} }
-  - match: { contexts.aggs_init: {} }
-  - match: { contexts.aggs_map: {} }
-  - match: { contexts.aggs_reduce: {} }
-  - match: { contexts.bucket_aggregation: {} }
-  - match: { contexts.field: {} }
-  - match: { contexts.filter: {} }
-  - match: { contexts.ingest: {} }
-  - match: { contexts.interval: {} }
-  - match: { contexts.number_sort: {} }
-  - match: { contexts.processor_conditional: {} }
-  - match: { contexts.score: {} }
-  - match: { contexts.script_heuristic: {} }
```

Comment:
We seem to have lost a lot of builtin contexts here?

Reply:
Unfortunately the change to list made performing the comparison after this differ in different contexts.  This will be addressed when I address your next comment.

Reply:
I've cut this test down to make it more reliable, now just checking that at least one context exists, it has a execute method with return value field.